docs/content/getting_started/index.md

```diff
@@ -181,15 +181,17 @@ git config --global user.email "username@domain.com"
 git config --global user.name "username"
 ```
 
-### JDK + Maven (If using Type Safe API)
+### JDK + Maven (If using Type Safe API with Smithy)
 
-The Type Safe API requires JDK >= 11 and Maven >= 3.8 in order to perform the required code generation.
+Type Safe API with [Smithy](https://smithy.io/2.0/) requires JDK >= 11 and Maven >= 3.8 in order to build and validate the Smithy model.
 
 To install these, follow the below links:
 
 - JDK: We recommend installing any Coretto version >= 11 https://docs.aws.amazon.com/corretto/
 - Maven: You can any version of Maven >= 3.8: https://maven.apache.org/install.html
 
+If you are using OpenAPI as your modelling language, these are not required.
```

Comment:
It might be slightly swapping how this information is presented: make these points an "info" (or some such) call out panel and lead with the point on line 193 (i.e. "Note: if you intend to use Smith…" blah blah blah).